### PR TITLE
Creating a stub of the file for Helen xmzhou

### PR DESCRIPTION
**Pull request by helenxmzhou**
Adding sub section for what we want to know about FV NC

### CODE REVIEW

File: helen-xmzhou.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Helen-xmzhou, CA
+
+### Great places to meet people
```
Comment (helenxmzhou) [inline helen-xmzhou.md:3]: add a comment line 3

Comment (barbkoz) [inline helen-xmzhou.md:3]: What are the top three places to meet people in your area?

Comment (helenxmzhou) [inline helen-xmzhou.md:3]: beach, park and hotel

Comment (barbkoz) [inline helen-xmzhou.md:3]: changes look good 👍

**Pull request merged**